### PR TITLE
Upgrade Jinja2

### PR DESCRIPTION
**Pull request by ubajze**

### CODE REVIEW

**Comment by ubajze** [comment]
@glennmatthews @PhillSimonds I would appreciate it if you can release a new patch for this project.
The `nautobot-gizmo` depends on the `nautobot-chatops-grafana` plugin and the the `nautobot-chatops-grafana` plugin depends on the `schema-enforcer`. 

This project had Jinja hardcoded to 2.x. But, Nautobot 1.3.x requires Jinja 3.x, so I cannot upgrade Nautobot in `nautobot-gizmo`.

File: pyproject.toml [hunk 1]
```diff
@@ -20,7 +20,7 @@ termcolor = "^1.1"
 jsonschema = "^3.2"
 toml = "^0.10"
 "ruamel.yaml" = "^0.16 || ^0.17"
-jinja2 = "^2.11"
+jinja2 = "*"
```
Comment (glennmatthews) [inline pyproject.toml:23]: Probably ">=2.11" would be safer?

**Comment by ubajze** [comment]
@glennmatthews good to merge and bump the version?

**Review by glennmatthews — APPROVED** [review]

**Review by PhillSimonds — APPROVED** [review]
LGTM.

**Pull request merged**